api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -159,4 +168,28 @@ private void validateStopDatetime(Visit visit, Visit otherVisit, Errors errors)
 		}
 		
 	}
+	
+	private void validateBirthDate(Visit visit, Errors errors) {
+		
+		if (visit.getPatient().getBirthdate() != null && visit.getStartDatetime() != null) {
```

Comment:
turn this into a "guard clause" to remove nesting and make it easier to read. by inverting the checks.
(already included the if from above when patient is null)

```
if (visit.getPatient() == null || visit.getPatient().getBirthdate() == null || visit.getStartDatetime() == null) {
  return;
}